package.json

```diff
@@ -183,35 +191,32 @@
     "deploy": "gh-pages -d styleguide && gh-pages -d styleguide -o upstream",
     "deploy:origin": "yarn install && yarn run predeploy && gh-pages -d styleguide && gh-pages -d styleguide -o origin",
     "watch:css": "yarn run build:css && sass src --watch --load-path=node_modules",
-    "start-js": "react-scripts start",
+    "start-js": "craco start",
     "start": "npm-run-all -p watch:css serve:docs",
     "start-deprecated-demo": "npm-run-all -p watch:css start-js",
-    "test": "react-scripts test --transformIgnorePatterns \"node_modules/(?!(lodash-es|jest*)/)\" , --env=jsdom --setupFilesAfterEnv ./src/setupTests.js --resetMocks false",
+    "test": "craco test --transformIgnorePatterns \"node_modules/(?!(lodash-es|jest*)/)\" , --env=jsdom --setupFilesAfterEnv ./src/setupTests.js --resetMocks false",
     "test:coverage": "yarn test --coverage",
     "test:noPrompt": "CI=true yarn test",
-    "eject": "react-scripts eject",
+    "eject": "craco eject",
     "type-check": "tsc --noEmit",
     "type-check:watch": "yarn run type-check --watch",
     "build": "yarn run build:css && yarn run build:js && yarn run build:copy-assets && yarn build:esbuild",
     "build:css": "sass src/lib/style/main.scss src/lib/style/main.css --load-path=node_modules && sass src/demo/style:src/demo/style --load-path=node_modules",
     "build:js": "tsc --p tsconfig.build.json",
     "build:copy-assets": "cd src/lib && find . -name '*.scss' -o -name '*.css' -o -name '*.svg' | cpio -pdm ../../dist/",
     "build:esbuild": "node esbuild.config.mjs && cp -r src/umd dist/",
-    "serve:docs": "DISABLE_ESLINT_PLUGIN=true FAST_REFRESH=false npx styleguidist server --open",
-    "build:docs": "yarn run build:css && DISABLE_ESLINT_PLUGIN=true FAST_REFRESH=false npx styleguidist build",
+    "serve:docs": "npx styleguidist server --open",
```

Comment:
Removed the FAST_REFRESH=false since I got fast refresh working...it's pretty cool, under certain conditions React components can update to reflect your code changes without reloading, even maintaining state after the refresh

Reply:
And as mentioned before, the DISABLE_ESLINT_PLUGIN is now part of the webpack config in craco.config.js